### PR TITLE
feat: implement the ruby module

### PR DESCRIPTION
**Pull request by AZanellato**
#### Motivation and Context

There isn't a ruby prompt, which is one more language for the prompt.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [x] I have tested using **Linux**
- [ ] I have tested using **Windows**

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [x] I have updated the tests accordingly.

### CODE REVIEW

File: src/modules/ruby.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Ruby version
+///
+/// Will display the Ruby version if any of the following criteria are met:
+///     - Current directory contains a `.rb` file
+///     - Current directory contains a `Gemfile` file
+///     - Current directory contains a `.ruby-version` file
```
Comment (matchai) [inline src/modules/ruby.rs]: I would drop support here for `.ruby-version` until we specifically support RVM.
```suggestion
```

File: src/modules/ruby.rs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let is_rb_project = context
+        .new_scan_dir()
+        .set_files(&["Gemfile", ".ruby-version"])
```
Comment (matchai) [inline src/modules/ruby.rs]:
```suggestion
        .set_files(&["Gemfile"])
```

File: src/modules/ruby.rs (hunk 1 above)
Comment (AZanellato) [reply src/modules/ruby.rs]: Done! :smile:

**Comment by matchai** [comment]
@all-contributors please add @AZanellato for code, docs and tests.

**Pull request merged**